app/styles/components/_users-pagination-links.scss

```diff
@@ -0,0 +1,14 @@
+.users-pagination {
```

Comment:
I took out `<div class="users-pagination">` from `user-pagination-links.hbs` and changed above to `.users-pagination-links`.

I guess I was slightly coding for the future. Components will eventually render only the contents inside it without the wrappers. When that happens, some type of wrapping div needs to inserted (same for lots of other components).